README.md

```diff
@@ -1,27 +1,28 @@
-# Docs Build - Local Validation
+# Docs Validation
 
 [![Build Status](https://dev.azure.com/ceapex/Engineering/_apis/build/status/Docs%20Validation/docascode.vscode-docs-build?branchName=master)](https://dev.azure.com/ceapex/Engineering/_build/latest?definitionId=1716&branchName=master)
 
-This is a VS Code extension to provide the Local Build Validation experience for a docs repository.
+This extension enables you to run build validation on a Docs conceptual or Learn repo at author time in VS Code. This means you can make sure your repo is free of validation issues before making a pull request.
 
-## Features
+## Prerequisites
 
-- Ability to sign in/out of the dependant Docs APIs, and display the status in the status bar
-- Validate the current workspace folder
-- Display the real-time build streaming output in the `Docs` channel
-- Display all diagnostics in the `Problem` view with detailed information after a build has finished.
-![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
+- You must be a Microsoft employee.
```

Comment:
So we are going to remove this one after we make it public for outside contributor, right? it will be soon.

Reply:
Yes, we can update the readme for outside contributors now as long as we don't ship it with misleading information.

Reply:
We will release a new version extension with this update and supporting the outside contributor. let's do that.

Reply:
Updated to remove requirement to be a Microsoft employee.